pkg/forwarder/transaction/transaction.go

```diff
@@ -192,6 +199,9 @@ type HTTPTransaction struct {
 	CompletionHandler HTTPCompletionHandler
 
 	Priority Priority
+
+	// In Serverless mode, requests should not be reused on failure retries
```

Comment:
I don't think it's the **request** that is being reused, but rather the **connection**.

```
// Close indicates whether to close the connection after
// replying to this request (for servers) or after sending this
// request and reading its response (for clients).
```

Reply:
true, I'll correct the comment

Reply:
Also, per Go convention I would start this comment with `CloseAfterCreation` so that it gets picked up as a docstring by VSCode/IDEs